main.py

```diff
@@ -2,49 +2,49 @@
 
 def path_to_file_list(path: str) -> List[str]:
     """Reads a file and returns a list of lines in the file"""
-    li = open(path, 'w')
-    return lines
+    lines = open(path, 'r').readlines()
+    return [line.strip() for line in lines]
```

Comment:
Reason: Changed from write (`'w'`) to read (`'r'`) mode to correctly read the file lines.